### ISSUE TITLE
Hitting home button should bring you to your profile page

### ISSUE THREAD

**Issue by ARealDuck** [Reyder95/sdvx-tracker-frontend#117]
the home page should be a welcome page instead of what it is and in place it should be a button to bring you to your profile

some players have confusion with the button and thus I think its a good idea to change it

**Comment by Malikil** [comment]
I agree, the current "home" page feels more like a landing page for people who don't currently have an account. When you're logged in the main page you'd generally be looking at would be your profile.  
You could say it might be the songs list instead, but imo the songs list is fine being accessed by clicking to it at the top.

**Comment by Reyder95** [comment]
Personally I think the home page can be fine being kept as "Home" ... however if you're accessing the site and are already logged in, then I can just redirect them to the profile instead.
